az-digital/az_quickstart
# Adding sidebar block to News issue/552

This PR adds the Promoted Sidebar block to the az_news view. It also creates a view mode/display for thumbnail to az_image

## Description
This ticket:
- Updates the az_news view with a new block display called "Sidebar News Block" to show stories three most recent stories
- Adds the view mode/display mode "Minimal Media List" that is used by the view
- Adds the view mode/display mode "Thumbnail" that is used by associated media for this view
- Places the block az_news_sidebar to the second sidebar region for pages /news/*

## Related Issue
#552 

## How Has This Been Tested?
Tested locally.

To test changes:
1. Create several news items
2. View a news item and verify that them most recent news items appear in the sidebar on the right
3. Verify that the article you are viewing does not appear in the sidebar stories

![image](https://user-images.githubusercontent.com/10873398/110371524-54764f00-800a-11eb-92fe-1a0b033d4f66.png)
![image](https://user-images.githubusercontent.com/10873398/110715716-00fb3100-81c3-11eb-9af6-75239cf8da72.png)


## Types of changes


## Checklist:
